pyro/distributions/zip.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.distributions import constraints
+from torch.distributions.utils import broadcast_all, lazy_property
+
+from pyro.distributions import TorchDistribution
+
+
+class ZIP(TorchDistribution):
```

Comment:
nit: I'm tempted to name this `ZeroInflatedPoisson` and let users `import ZeroInflatedPoisson as ZIP`

Reply:
+1 for more descriptive names, it's also consistent with the rest of the distribution names